xapian-core/backends/databaseinternal.cc

```diff
@@ -104,6 +104,20 @@ Database::Internal::get_wdf_upper_bound(const string & term) const
     return cf;
 }
 
+Xapian::termcount
+Database::Internal::get_uniq_terms_lower_bound() const
+{
+    throw Xapian::UnimplementedError("This backend doesn't support"
+			       "get_uniq_terms_lower_bound");
+}
```

Comment:
However, `1` is a valid lower bound.

Reply:
I will add this. I thought, no other backend support this interface, thought setting error makes it clear if called by chance.

Reply:
Done

Reply:
We want to be able to call these methods from `Xapian::Weight` subclasses without having to worry about whether the backend actually tracks these values or not.

Currently `DiceCoeffWeight` uses `1` as the lower bound, so returning 1 in the default implementation means that class can just ask the database for the lower bound, and we'll use the same value as currently for backends which don't track these bounds, and the tracked bound for backends which do track it.